portfolioStyles.css

```diff
@@ -0,0 +1,362 @@
+*{
```

Comment:
the css selectors should have a space before the { eg: * {
please change throughout this file